dags/run_all_processors.py

```diff
@@ -0,0 +1,43 @@
+import datetime
+from airflow.models import DAG
+from airflow.operators.latest_only_operator import LatestOnlyOperator
+import utils.helpers as helpers
+
+args = {
+    'owner': 'airflow',
+    'depends_on_past': False,
+    'start_date': datetime.datetime(2017, 1, 1),
```

Comment:
The DAG doesn't need a `start_date`, but the tasks do (see https://travis-ci.org/opentrials/opentrials-airflow/builds/220910597). Just added it back in.